examples/adaptive_library/DXFImport.py

```diff
@@ -0,0 +1,205 @@
+"""
+Custom DXF Geometry
+===================
+This script applies the adaptive templates functionality to import custom rotor geometry from a
+.dxf file.
+"""
+# %%
+# .. note::
+#    For more information on how to add imported DXF geometries to Adaptive Templates in Motor-CAD,
+#    see :ref:`ref_adaptive_templates_UG` in the :ref:`ref_user_guide`.
+#
+# .. note::
+#    DXF Import for Adaptive Templates in Motor-CAD requires v2024.2.0.6 (Motor-CAD 2024 R2) or
+#    later and PyMotorCAD v0.4.2. To update PyMotorCAD in Motor-CAD, go to
+#    **Scripting -> Settings** and select 'Update to Latest Release' under PyMotorCAD updates.
```

Comment:
```suggestion
#    DXF import for Adaptive Templates in Motor-CAD requires v2024.2.0.6 (Motor-CAD 2024 R2) or
#    later and PyMotorCAD v0.4.2. To update PyMotorCAD in Motor-CAD, select
#    **Scripting > Settings**. Under **PyMotorCAD updates**, select **Update to Latest Release**.
```
All content formatted in bold is assumed to be UI elements and that the case matches what is shown in the UI.